pkg/controllers/management/feature/feature_handler.go

```diff
@@ -40,9 +43,25 @@ func (h *handler) sync(_ string, obj *v3.Feature) (*v3.Feature, error) {
 	if obj.Name == features.TokenHashing.Name() {
 		return obj, h.refreshTokens()
 	}
+
+	if obj.Name == features.Harvester.Name() {
+		return obj, h.toggleHarvesterNodeDriver(obj.Name)
+	}
 	return obj, nil
 }
 
+func (h *handler) toggleHarvesterNodeDriver(harvester string) error {
+	lister := h.mgmt.Management.NodeDrivers("").Controller().Lister()
+	cli := h.mgmt.Management.NodeDrivers("")
```

Comment:
Are empty string inputs for the ManagementContext NodeDrivers always going to work? We may also be able to use one NodeDrivers interface and create another one for the lister rather than calling it twice.

Reply:
This is what I was getting at for the second point: https://github.com/rancher/rancher/pull/35315#discussion_r743008038

Reply:
@nickgerace I just updated the code. Many thanks for the input.